models/repo.go

```diff
@@ -729,9 +729,9 @@ func (repo *Repository) getReviewersPublic(e Engine, doerID, posterID int64) (_
 	return users, nil
 }
 
-func (repo *Repository) getReviewers(e Engine, doerID, posterID int64) (users []*User, err error) {
+func (repo *Repository) getReviewers(e Engine, doerID, posterID int64) (users []*User, teams []*Team, err error) {
```

Comment:
Please rewrite this function with such signature as this currently can happen to ignore and lose error on L739 or L741
```suggestion
func (repo *Repository) getReviewers(e Engine, doerID, posterID int64) ([]*User, []*Team, error) {
```